terraform/deployments/monitoring-test/grafana/app-config/grafana_config_terraform_results

```diff
@@ -0,0 +1,9 @@
+[0m[1mdata.terraform_remote_state.monitoring: Refreshing state...[0m
```

Comment:
Did you mean to commit this file?

Reply:
nope, good catch. It was more for debugging.